impactmass/symmetrical-octo-garbanzo
# feat: new uri; payment and subaccount options; redirect fix from Rave

## Changes
- Added support for Cordova in-app browser and using redirect URL for 3DSecure payments.
- Added support for recurring payments, split payments.

## Breaking changes
N/A

## Testing
- Confirm that OTP Modal issue is fixed.
- Confirm all previous use cases work as before
